### PR TITLE
debt: states

### PR DESCRIPTION
**Pull request by GabrielPrediger**

### CODE REVIEW

File: source/pages/SwitchNetwork/NetworkList.tsx [hunk 1]
```diff
@@ -53,7 +52,7 @@ export const NetworkList = ({ isChanging }: { isChanging: boolean }) => {
 
   const handleChangeNetwork = async (network: INetwork, chain: string) => {
     try {
-      store.dispatch(setOpenDAppErrorModal(false));
+      wallet.setOpenDAppErrorModalState();
```
Comment (lucaslovato) [inline source/pages/SwitchNetwork/NetworkList.tsx:55]: is this change correct?

Comment (GabrielPrediger) [reply source/pages/SwitchNetwork/NetworkList.tsx:55]: As we said in the chat, yes, the function that is marked is correct and does not expect any arguments

Comment (Potolski) [reply source/pages/SwitchNetwork/NetworkList.tsx:55]: It is not correct. You actually created a function that already existed. Previously we were passing **false** as the argument of the function and now you are passing true.

File: source/scripts/Background/controllers/MainController.ts [hunk 2]
```diff
@@ -1648,6 +1652,7 @@ const MainController = (walletState): IMainController => {
     ...keyringManager,
     openDAppErrorModal,
```
Comment (Potolski) [inline source/scripts/Background/controllers/MainController.ts:1653]: This is the method I mentioned https://github.com/syscoin/pali-wallet/pull/602/files#r1695634367

**Pull request closed without merging**